src/Dialogs/DataField.cpp

```diff
@@ -57,6 +58,12 @@ EditDataFieldDialog(const TCHAR *caption, DataField &df,
 
     gdf.ModifyValue(value);
     return true;
+  } else if (df.GetType() == DataField::Type::NUMBER) {
```

Comment:
Doesn't compile - `DataField::Type::NUMBER` doesn't exist.

Reply:
The type NUMBER is removed now, it is a INTEGER yet